### ISSUE TITLE
mypy hook ignores `--exclude` directive

### ISSUE THREAD

**Issue by ethanabrooks** [pre-commit/mirrors-mypy#64]
I have the following `.pre-commit-config.yaml`:
```yaml
repos:
  - repo: https://github.com/pre-commit/mirrors-mypy
    rev: v0.931
    hooks:
      - id: mypy
        args: [--exclude=fails.py]
```
this poorly typed file:
```python
# fails.py
x: str = 1
```
and this well-typed file:
```python
# succeeds.py
x: str = "1"
```

When I run `pre-commit run mypy --all-files` or I commit, I get:
```
mypy.....................................................................Failed
- hook id: mypy
- exit code: 1

fails.py:1: error: Incompatible types in assignment (expression has type "int", variable has type "str")
Found 1 error in 1 file (checked 1 source file)
```
Meanwhile this works:
```
❯ mypy --exclude=fails.py .
Success: no issues found in 1 source file
```

**Comment by asottile** [comment]
try [searching the issue tracker next time](https://github.com/pre-commit/mirrors-mypy/issues?q=is%3Aissue+--exclude+is%3Aclosed+)

perhaps [this](https://github.com/pre-commit/mirrors-mypy/issues/24#issuecomment-663594662)
